specification/resources/resource-manager/Microsoft.Resources/2015-11-01/resources.json

```diff
@@ -1090,6 +1090,86 @@
           }
         }
       },
+      "patch": {
+        "tags": [
+          "Resources"
+        ],
+        "operationId": "Resources_Update",
+        "description": "Updates a resource.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
```

Comment:
in case you have bandwidth, I'd strongly suggest factoring out common parameters like this one into the `parameters` section. Much deduplication! 🙂 In that case, don't forget to add `x-ms-parameter-location: method` so the parameter stays on the method and doesn't become a client property

Reply:
Understood - I will leave refactoring to the resources team.